packages/turf-polygonize/test.js

```diff
@@ -0,0 +1,51 @@
+const test = require('tape'),
+  { featureCollection, lineString, polygon } = require('@turf/helpers'),
+  polygonize = require('./'),
+  fs = require('fs'),
+  path = require('path');
+
+const directories = {
+  in: path.join(__dirname, 'test', 'in') + path.sep,
+  out: path.join(__dirname, 'test', 'out') + path.sep,
+};
+
+function getFullPath(filename, type='in') {
```

Comment:
Don't know the purpose of this? Since the files are being saved to `test/out` you can simply use:

```javascript
directories.out + filename
```